src/queries/GetProject/getProject.js

```diff
@@ -0,0 +1,21 @@
+import {gql} from 'urql';
```

Comment:
`src/queries/GetProject/getProject.js`
`GetProject` should be replaced with feature name